R/set_pc_rownames.R

```diff
@@ -0,0 +1,23 @@
+#' Label rownames of provided PCs and remove cells with NA batch label from PCs
+#'
+#' @param pcs The PCs with unlabeled rownames
+#' @param batches Vector of cell-wise batch information whose length is the number of
+#'   rows in `pcs`
+#'
+#' @return PCs with labeled rownames and NA batch cells removed
+set_pc_rownames <- function(pcs, batches) {
```

Comment:
I think there was some discussion of the naming of this function, but since it also filters based on batch presence, I might rename it to something that reflects that. maybe `filter_pcs`. The fact that it adds names almost seems secondary to the filtering?